## Amplification
- Many RNA copies can be made from a single gene, and many proteins can be made from one RNA copy
	- This *amplification* allows for a lot of protein to be made quickly
## RNA
- *Transcription*: [[The structure of DNA#DNA Consists of Two Complementary Chains of Nucleotides|nucleotide sequence]] is copied onto *RNA*
- *RNA* is much like *DNA*, but single stranded, has rubonicleotides, and instead of T it has *U, Uracil*
	- *DNA and RNA* still differ a lot in structure --> it can fold more
	- Therefore, RNA can also have other uses than info store
## Transcription
- Process of making RNA from DNA
- One of the two strands of DNA acts as *template*
	- Nucleotides are added to the RNA chain by *RNA polymerase*
	- This happens in the 5'-to-3' direction
	- *step-by-step*, just like DNA Replication
- Differences Transcription and DNA replication:
	- RNA does not stay connected to the DNA --> single stranded
	- RNA molecules are much shorter
	- Uses *ribonucletides*, not *deoxyribonucleotides*
	- RNA transcription can start without a primer
	- RNA makes a mistake every 10<sup>4</sup> nucleotides, DNA only one every 10<sup>7</sup>
- The next RNA molecule already started being produced before the previous one finishes
- A gene of 1500 base pairs takes 50 seconds to transcribe
## Different types of RNA
- Most genes specify the amino acid sequence of [[How proteins work|proteins]]
	- RNA that carries this info are called *messenger RNA, or mRNA*
		- In eukaryotes only one gene per mRNA, in bacteria often more. 
- Sometimes the final product is the RNA
	- Various roles, regulatory, structural, and catalytic components
	- [[Table of RNA types]]
## Promotor & terminator
- When *RNA polymerase* randomly encounters DNA, it will stick weakly to the DNA
	- It will find a *promotor sequence* in a gene, and bind
	- This opens up the double helix, and starts the transcription
	- Will continue until it finds a *terminator* --> stops transcription
- [[Transcription figure]] -> explains it quite well
- How does the *RNA polymerase* recognize the *promotor*?
	- each base presents unique features to the outside of the double helix, allowing the *sigma factor* to find the promoter sequence without having to separate the entwined DNA strands
- Which DNA strand to use as a *template*?
	- The *promotor* has a polarity: Two different nucleotide sequences upstream of the transcriptional start site that position the *RNA polymerase*
		- Ensures it only binds in one orientation
		- As RNA polymerase can only read from 3'-to-5', only one direction is possible -> the right one!
### Initiation of gene transcription in eukaryotes
- Above was about prokaryotes, but eukaryotes works mostly similarly
- Differences:
	- Eukaryotic cells have multiple types of RNA polymerase.
		- ![[Pasted image 20240930163932.png|300]]
	- Eukaryotic cells need many accessory proteins, called *transcription factors* which assemble at a promotor
	- Eukaryotic DNA has more space between the genes for *regulatory DNA sequences*
	- Eukaryotic DNA is compacted into [[The structure of eukaryotic chromosomes#Nucleosome|Nucleosomes]]
### General transcription factors
- They assemble on the *promotor* and pull apart the double helix
	- Thus *transcription* can start
	- Similar to *sigma factor* in bacteria
- [[Example transcription factors]]
	- This one is about TATA box, which starts 15 nucleotides before promotor
	- RNA polymerase liberation is done after addition of phosphate groups to its tail
- Once transcription has started, the transcription factors are released
	- When transcription has finished, the phosphate group is removed and the polymerase released, ready to find another promotor (can only initiate transcription unphosphorylated)
### The nucleus
- in Eukaryotic cells, DNA enclosed within the *nucleus*
	- *Transcription* takes place in nucleus, but protein synthesis happens outside
	- therefore, mRNA is transported through small pores in the nucleus (*nuclear pore complex*) --> very *selective, can only be done after all the RNA processing is done*
- Before this can happen, *RNA processing* must first occur
	- *Capping, splicing, and polyadenylation* that mark the RNA as *mRNA*.
#### Capping
- straight up lying to somebody
- Modifies the 5' end of RNA, *the end that is synthesized first*
	- adds a really weird *Guanine* nucleotide with methyl group
	- Happens after 25 nucleotides have been added (quickly)
#### Polyadenylation
- First, 3'-end is trimmed, then a few hundred A (*adenine*) nucleotides are added
### Introns
- Introns are non-coding, *intervening sequences* in genes within genes. The coding sequences are called *exons*
- They are removed from pre-mRNA by *RNA splicing*
	- [[RNA splicing figure]]
	- Exons are stitched together
- Each intron has a few nucleotide sequences that act as cues for splicing
	- Splicing done by *small nuclear RNAs* (snRNAs)
		- Form core of *spliceosome*
- Isnt this wasteful? --> No, it has benefits:
	- *Alternative splicing* -> making many different proteins from one gene
	- Allows more, different proteins to be made during evolution
		- Many proteins are a patchwork of different protein set pieces --> [[The Shape and Structure of proteins#Protein domain|protein domains]]
## mRNA degradation
- Once in the cytosol, the mRNA is degraded
- The less common the protein is, the shorter its life span
	- If it is not common, the protein is probably only needed given specific signals, so not good to keep producing more
- ranges from 30 minutes to 10 hours, bacteria mostly only 3 minutes
